src/backend.cpp

```diff
@@ -6,14 +6,14 @@ See https://github.com/vpisarev/loops/LICENSE
 
 #include "backend.hpp"
 #include "func_impl.hpp"
-
+#include <iostream>
 namespace loops
 {
-Mnemotr::Argutr::Argutr(const Arg& a_fixed) : tag(T_FIXED), fixed(a_fixed) {}
-Mnemotr::Argutr::Argutr(size_t a_src_arnum) : tag(T_FROMSOURCE), srcArgnum(a_src_arnum) {}
+Mnemotr::Argutr::Argutr(const Arg& a_fixed) : tag(T_FIXED), fixed(a_fixed), srcArgnum(-1), transitFlags(0) {}
+Mnemotr::Argutr::Argutr(size_t a_src_arnum, uint64_t flags) : tag(T_FROMSOURCE), srcArgnum(a_src_arnum), transitFlags(flags) {}
 Mnemotr::Mnemotr(int a_tarop, std::initializer_list<Argutr> a_args) : m_tarop(a_tarop), m_argsList(a_args){}
 
-Syntop Mnemotr::apply(const Syntop& a_source, const BackendImpl* a_bcknd) const
+Syntop Mnemotr::apply(const Syntop& a_source, const Backend* a_bcknd) const
```

Comment:
please, do not compress backend to bcknd. I've seen m_bcknd and many other such elements. It's quite difficult to read and to type. Expand it to "backend". This also relates to many other shortcuts. The general rule of thumb - if you are going to throw away 1-2 letters from the word, don't do it.

Reply:
Done.